js/viewer/annotation/base.js

```diff
@@ -78,6 +212,14 @@ Base.prototype.renderSVG = function(){
             .on("mousemove", this.onMousemoveShowTooltip)
             .on("mouseout", this.onMouseoutHideTooltip)
             .on('click', this.onClickToSelectAnnotation);
+            
+        // prevent the default behavior of osd by adding an empty click handler
+        new OpenSeadragon.MouseTracker({
```

Comment:
I know I added this, but I didn't know we have an `unbind` function. I think we should capture this object and destroy it in the unbind function, so

```javascript
this._defaultClickHandler = new OpenSeadragon.MouseTracker({
  element: this.svg.node(),
  clickHandler: function (e) {
    // intentially left empy. we just want to prevent the default behavior (zoom)
  }
});

```

Then in the `unbind` we should just do 

```
if (this._defaultClickHandler) {
  this._defaultClickHandler.destroy();
}
```
(we should test and make sure this works properly)